ci/cfg/ci_kmac_sca_random_cw310_simpleserial.yaml

```diff
@@ -33,7 +33,7 @@ husky:
   scope_gain: 26
 capture:
   scope_select: husky
-  num_traces: 1000
+  num_traces: 100
```

Comment:
Is this reduction in number of traces on purpose or accidental?

Reply:
Although it is not related to this PR, I noticed that capturing 1000 traces in the non-batch mode takes quite some while. As we do not need to capture that many traces, I suggest to reduce it to 100 traces.

Reply:
I agree it's a good idea to reduce it to 100. Some captures run at ~10 traces/s so capturing 1000 traces significantly slows down CI

Reply:
I've moved this into a separate PR #298